doc/api/http2.md

```diff
@@ -1064,7 +1064,7 @@ const server = http2.createServer();
 server.on('stream', (stream) => {
   function statCheck(stat, headers) {
     // Check the stat here...
-    stream.respond({ ':status': 304 });
+    stream.respond({ ':status': 304 }, { endStream: true });
```

Comment:
Maybe I misread the example but isn't this code snippet showing a cancellation of the file transfer and instead responding with a Not Modified header frame? The stream remains open unless the endStream flag is set.